src/Features/CSharp/Portable/PassVariableExplicitlyInLocalStaticFunction/PassVariableExplicitlyInLocalStaticFunctionCodeFixProvider.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.GetCapturedVariables;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Roslyn.Utilities;
+
+namespace PassVariableExplicitlyInLocalStaticFunction
```

Comment:
please follow the namespace naming convention in the repo. e.g. `Microsoft.CodeAnalysis.CSharp.MakeLocalFunctionStatic`